### ISSUE TITLE
It would be nice to warn when using "ancestor_id" that the page isn't found and should be created

### ISSUE THREAD

**Issue by seburdin** [dauxio/daux.io#352]
**Is your feature request related to a problem? Please describe.**
I tried to use the ancestor_id feature for confluence publishing but got the error

```
Finding Root Page...

In Publisher.php line 125:
                                                                               
  You must at least specify a `root_id` or `ancestor_id` in your confluence c  
  onfiguration.                                                                
```

**Describe the solution you'd like**
Throw an exception here : https://github.com/dauxio/daux.io/blob/5190743428688b8de02cb7895eea4cfc3f95ec96/libs/Format/Confluence/Publisher.php#L116
to say the page wasn't found and should be created manually

**Comment by seburdin** [comment]
After creating the ancestor page  manually, ancestor tag still doesn't work and shows the same error in Publisher.php line 125. Only root_id tag seems to work.

**Comment by onigoetz** [comment]
This change has been released in [0.20.1](https://github.com/dauxio/daux.io/releases/tag/0.20.1)
